src/lib/chips/chips.scss

```diff
@@ -77,3 +89,33 @@ $mat-chips-chip-margin: $mat-chip-horizontal-padding / 4;
     }
   }
 }
+
+$default-remove-icon-offset: 3px;
+
+.mat-chip-remove {
+  position: absolute;
+  top: 2px;
+  right: 4px;
+  width: 24px;
+  padding-top: $default-remove-icon-offset;
+  height: 24px - $default-remove-icon-offset;
+  font-size: 18px;
+  text-align: center;
+  cursor: default;
+}
+
+.mat-chip-remove.mat-chip-remove-hidden {
+  display: none;
+}
+
+// Override a few styles when inside an mat-input-container
+.mat-input-container .mat-chip-list-wrapper input {
+  width: auto;
+  height: 38px;
+  margin-left: 8px;
```

Comment:
Is there a way we can move those hard-coded values to variables?

And above for `.mat-chip-remove`, couldn't we calculate those `left`, `right` values?